CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 2.8)
 
+option( BUILD_EXAMPLE "Build example application."                  ON )
```

Comment:
Let's be consistent with coding style:
- no spaces before and after parentheses
- no long horizontal spacing